### ISSUE TITLE
fantasy.formula1.com: ads

### ISSUE THREAD

**Issue by Trilantini** [uBlockOrigin/uAssets#12317]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://fantasy.formula1.com/app/`

### Category

ads

### Describe the issue

An element which is called "flt-platform-view" on the fantasy f1 website, is a sort of framework that stores ads location. Ads in there aren't shown but the element is still there, and it's kind of annoying

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![wKlr3qex8Q](https://user-images.githubusercontent.com/43372972/159012565-ae632219-4370-490f-b018-32797ec3896a.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.41.8
Chromium: 99
filterset (summary): 
  network: 153069
  cosmetic: 105157
  scriptlet: 33951
  html: 0
listset (total-discarded, last updated): 
  added: 
    https://raw.githubusercontent.com/Spam404/lists/master/adblock-list.txt: 7253-136, 2d.37m
    https://winhelp2002.mvps.org/hosts.txt: 8730-2, 5d.1h.5m
    adguard-annoyance: 50869-1681, 2d.21h.16m
    adguard-generic: 62733-1076, 22h.57m
    adguard-mobile: 7353-111, 22h.59m
    adguard-spyware: 26761-1132, 22h.58m
    adguard-spyware-url: 513-12, 1d.2h.14m
    block-lan: 43-0, 5d.20h
    curben-phishing: 10726-0, 1h.26m
    curben-pup: 333-0, 1h.24m
    dpollock-0: 10347-1473, 5d.19h.58m
    fanboy-cookiemonster: 25276-14991, 2d.21h.17m
    ublock-annoyances: 4367-284, 1d.2h.10m
  default: 
    user-filters: 185-6, never
    easylist: 64273-24943, 22h.56m
    easyprivacy: 26702-10648, 22h.55m
    plowe-0: 3671-1483, 2d.38m
    ublock-abuse: 72-0, 22h.54m
    ublock-badware: 3922-254, 13h.16m
    ublock-filters: 30355-3658, 22h.53m
    ublock-privacy: 200-34, 22h.52m
    ublock-unbreak: 1740-114, 22h.51m
    urlhaus-1: 7983-1, 1h.25m
filterset (user): [array of 185 redacted]
trustedset: 
  added: [array of 13 redacted]
  removed: 
    edge-scheme
switchRuleset: 
  added: [array of 5 redacted]
modifiedUserSettings: 
  uiAccentCustom: true
  uiAccentCustom0: #00d6bd
  uiTheme: dark
  ignoreGenericCosmeticFilters: true
  largeMediaSize: 90
  showIconBadge: false
  suspendUntilListsAreLoaded: true
  webrtcIPAddressHidden: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 15509 ms
  maxAssetCacheWait: 1526 ms
popupPanel: 
  blocked: 2
  blockedDetails: 
    googletagmanager.com: 1
    googletagservices.com: 1
```
</details>

**Comment by mapx-** [comment]
account needing: use the element picker, create a filter and post it here

**Comment by stephenhawk8054** [comment]
I have tested with a burner email, looks like it's in `shadow-root`, so might be not fixable

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/66517106/159102256-6697a090-7333-4762-bef8-f1dcbfb5433b.png)

</details>

**Comment by mapx-** [comment]
the shadow-root is gone using
`formula1.com##+js(set, HTMLElement.prototype.attachShadow, null)`

**Comment by stephenhawk8054** [comment]
> the `shadow-root` is gone using `formula1.com##+js(set, HTMLElement.prototype.attachShadow, null)`

Ah right, confirm that works 👍 But I don't know how to hide that part, this site's DOM looks weird to me

**Comment by mapx-** [comment]
yes, indeed, idk either, it's some `MutationObserver` too

**Comment by Trilantini** [comment]
> account needing: use the element picker, create a filter and post it here

Yeah, with element picker it doesn't work. It will pick just this `##flt-glass-pane` which will basically hide everything from the site. Based on the website code that I looked at, it's really weird, I mean never saw that kind of practice where the whole site is only a few lines of code and probably the visual side is from a server or something like that. Tried even with this `fantasy.formula1.com/app/##flt-platform-view:nth-of-type(2)` or `... nth-of-type(1)`
![20Hco7N3mQ](https://user-images.githubusercontent.com/43372972/159126659-8703e62b-6a03-4f0c-95e0-9bb16589b8b7.png)

also, nothing will happen
